### DISCUSSION TITLE
Any intention to provide linux Processor driver to allow use on Raspberry Pi (not Pico)

### DISCUSSION THREAD

**Discussion by SteveSpencer** [Bodmer/TFT_eSPI#1645] [New processor/display support or capability request] [answered]
I am a big fan of TFT_eSPI on ESP hardware, and would like to use it on Raspberry Pi to drive SPI displays from userspace, rather than hope that kernel drivers become available for various TFT displays.

Any plans for this, or am I on my own?

Steve Spencer

**Comment by Bodmer** [comment, marked as answer]
The library relies on the Arduino support environment and board packages. It is possible to run sketches on a RPi according to the following link but I have not tried this:
https://www.deviceplus.com/raspberry-pi/how-to-run-arduino-sketches-on-raspberry-pi/
